web/themes/custom/cloudy/templates/content/construction_project/node--construction-project.html.twig

```diff
@@ -7,44 +7,34 @@
         {{ content.field_summary.0 }}
       </p>
       {{ content.field_featured_media }}
-      <div class="row">
-        <div class="col">
-          <div class="event__date">
-            <span class="d-flex align-items-center">
-            {% if content.field_display_date|render|striptags|trim is not empty %}
-              {{ content.field_display_date }}
-            {% else %}
-              {% if content.field_end_date|render|striptags|trim is not empty %}
-                {{ content.field_start_date }}
-                <span class="datetime mx-1" aria-label="{{ 'through'|t }}">&ndash;</span>
-                {{ content.field_end_date }}
-              {% else %}
-                {{ content.field_start_date }}
-              {% endif %}
-            {% endif %}
-            </span>
-          </div>
-          <div class="event_time">
-            <span class="d-flex align-items-center">
-              {% if content.field_all_day|render|striptags|trim is not empty %}
-                {{ content.all_day }}
-              {% elseif content.field_end_time|render|striptags|trim is not empty %}
-                {{ content.field_start_time }}
-                <span class="datetime mx-1" aria-label="{{ 'to'|t }}">&ndash;</span>
-                {{ content.field_end_time }}
-              {% else %}
-                {{ content.field_start_time }}
-              {% endif %}
-            </span>
-          </div>
-        </div>
-        {% if content.field_location %}
-        <div class="col-md-6 col-sm-12">
-          {{ content.field_location }}
-        </div>
+      <div class="event__date">
+        <span class="d-flex align-items-center">
+        {% if content.field_display_date|render|striptags|trim is not empty %}
```

Comment:
@joshuami What you have above will always override any set project dates if a value exists in field_display_date, disregarding the checkbox. Use the display date toggle value as the check to determine whether or not to show the display date.

Reply:
You could even check the toggle = true and display date is not empty.

Reply:
That's a good idea. I'll write it up as a follow up since we have the same pattern on the project group type and this wasn't s something I touched on this story.